elements/pfelement/src/pfelement.js

```diff
@@ -905,7 +911,7 @@ class PFElement extends HTMLElement {
   static _convertSelectorsToArray(selectors) {
     if (selectors) {
       if (typeof selectors === "string") return selectors.split(",");
-      else if (typeof selectors === "array" || typeof selectors === "object") return selectors;
+      else if (typeof selectors === "object") return selectors;
```

Comment:
@mwcz I think your other PR leaked into this one too :)

Reply:
Oops!  Thanks, good catch.  `git worktree add` didn't reset like I expected it to. :sweat_smile:

Reply:
Fixed.